server/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,4 @@
+server:
+  ssl:
+    key-store: classpath:test-keystore.p12
```

Comment:
I've generated self-signed certs for test which are okay to be added to the repository, as they are just to allow the test suite to run from any machine.